pkg/buildscript/raw.go

```diff
@@ -71,5 +71,5 @@ type null struct {
 
 type funcCall struct {
 	Name      string   `parser:"@Ident"`
-	Arguments []*value `parser:"'(' @@ (',' @@)* ','? ')'"`
+	Arguments []*value `parser:"'(' (@@ (',' @@)* ','?)? ')'"`
```

Comment:
This is basically saying that a function call can have zero or one argument list (consisting of 1 or more arguments).